src/modules/apiActions.js

```diff
@@ -0,0 +1,33 @@
+import { getScores, addScore } from './api';
+import displayElement from './displayElements';
+import {
+  userInput, scoreInput, submitBtn, tableBody,
+} from './Variables';
+import setFromIndicator from './indicators';
+
+export const refresh = async () => {
+  tableBody.innerHTML = 'LOADING SCORES...';
+  const scores = await getScores();
+  displayElement(scores);
+};
+
+export const addNewScore = async (e) => {
+  e.preventDefault();
+  const user = userInput.value;
+  const score = scoreInput.value;
+  if (user.trim() === '' || score.trim() === '') {
+    setFromIndicator('error', 'Please all fileds are required.');
+  } else {
+    submitBtn.value = 'Adding score...';
+    try {
+      const res = await addScore({ user, score });
+      setFromIndicator('success', res.result);
+      userInput.value = null;
+      scoreInput.value = null;
+      submitBtn.value = 'Add score';
+      refresh();
```

Comment:
[OPTIONAL]

Hello @AsumVictor. I have reviewed your project and I have a suggestion for you regarding the scores list. Currently, you are refreshing the scores list every time a new score is submitted. This might cause some performance issues and unnecessary requests to the API. A better approach would be to refresh the scores list only when the user clicks on the refresh button. This way, you can avoid making too many requests and also give the user more control over when they want to see the updated scores. This would also improve the user experience and make your app more responsive.

This suggestion is optional and not a requirement for this project, but I think it would be a nice improvement to your app. You can implement this suggestion by adding an event listener to the refresh button and calling the function that fetches and displays the scores inside it. You can also remove the call to that function from the submit button event listener.

I hope this suggestion helps you improve your project and learn more about web development. Keep up the good work!

Reply:
Thank you very much! I have removed the auto-refreshing feature.